v/vertx-json-schema/vertx-json-schema_ubi_8.5.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package	: vertx-json-schema
+# Version	: 4.0.1
+# Source repo	: https://github.com/eclipse-vertx/vertx-json-schema
+# Tested on	: ubi 8.5
+# Language      : node
```

Comment:
looks like this is java package, not node @adilhusain-s ?

Reply:
thanks for pointing out the typo, made the requested change.